templates/default/app-nginx.erb

```diff
@@ -1,4 +1,7 @@
+server_names_hash_bucket_size 128;
```

Comment:
Why is this needed?

Reply:
ill remove and see if the issue persists, but a while ago I was getting an error because that variable was too small, It might have been fixed upstream now though

Reply:
seems like this is still nesecary